### PR TITLE
Feature/cafe search

### PR DESCRIPTION
**Pull request by lili2311**
### ISSUE NUMBER

[Issue-43](https://github.com/womenhackfornonprofits/we-meet-django/issues/43)
### Describe the changes

Fixed static files not being picked up by django
Added the search field on the homepage
### Screenshots (if appropriate)

screen shot 2016-08-15 at 21 35 26

### CODE REVIEW

File: frontend/sass/_forms.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+select,
+textarea {
+	//display: block;
+    width: 100%;
+    margin: .5em 0;
+	padding: 1em;
+	border: 1px solid #ccc;
+	line-height: 1em;
+}
+
```
Comment (BibianaC) [inline frontend/sass/_forms.scss:16]: Indentation is inconsistent, could you fix it?

File: frontend/sass/_helpers.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	display: none;
+}
+
+.visually-hidden {
+	height: 0;
+    width: 0;
+    line-height: 0;
+    color: transparent;
+    background: transparent;
+}
```
Comment (BibianaC) [inline frontend/sass/_helpers.scss:11]: Indentation is inconsistent, could you fix it?

File: london_cafes/london_cafes/templates/index.html [hunk 3]
```diff
@@ -3,8 +3,13 @@
 {% block title %}Home{% endblock %}
 
 {% block main %}
-    <div>
-      <h1>London Cafes</h1>
-      <h2>Coming soon</h2>
-    </div>
+    <section class='container'>
+     	<h1>London Cafes</h1>
+      	<h2>Find a great place to work</h2>
```
Comment (BibianaC) [inline london_cafes/london_cafes/templates/index.html:8]: We might want to change the title `London Cafes`to the new `We meet`?

**Comment by lili2311** [comment]
Thanks @BibianaC, updated the PR

**Pull request merged**